### ISSUE TITLE
Comments autoCloseDays setting not respected on secondary sites

### ISSUE THREAD

**Issue by adrienne** [verbb/comments#140]
### Description

I have a multisite installation and while almost everything works correctly, the autoCloseDays setting in my config file is **not** respected on secondary sites. 

### Steps to reproduce

1. Set up a second site in Craft
2. Set up a `comments.php` config file with the `autoCloseDays` setting set to anything -- i use 14 days
3. Add the comment form to the secondary site (i'm using the following code:) 
```
{{ craft.comments.render(entry.id, {'orderBy': 'commentDate'}) }}
```
4. Check your older entries; the comment form will still be active.

### Additional info

- Plugin version: 1.1.10
- Craft version: Craft Pro 3.3.13

**Comment by adrienne** [comment]
This one, however, is definitely real.

**Comment by engram-design** [comment]
Should be fixed in [1.2.0](https://github.com/verbb/comments/releases/tag/1.2.0).
